src/Foundation/Enum.cs

```diff
@@ -498,15 +498,16 @@ public enum NSEnumerationOptions : ulong {
 		Reverse = 2
 	}
 	
-#if MONOMAC
+#if MONOMAC || __MACCATALYST__
 	[Native]
```

Comment:
Have you made sure it is present in the right version of Catalyst? Does it not need the min like the other ones we have seen?

Reply:
Not sure why this got missed. Fixed.